functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/failed/LocalChangesFailedTestsSelectionExecutionFunctionalTest.java

```diff
@@ -0,0 +1,64 @@
+package org.arquillian.smart.testing.ftest.failed;
+
+import java.nio.file.Path;
+import java.util.List;
+import org.arquillian.smart.testing.ftest.testbed.project.Project;
+import org.arquillian.smart.testing.ftest.testbed.rules.GitClone;
+import org.arquillian.smart.testing.ftest.testbed.rules.TestBed;
+import org.arquillian.smart.testing.ftest.testbed.testresults.TestResult;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+import static org.arquillian.smart.testing.ftest.failed.SurefireReportHandlingUtils.copySurefireReportsFromPreviousBuild;
+import static org.arquillian.smart.testing.ftest.failed.SurefireReportHandlingUtils.storeSurefireReportsForFailingBuild;
+import static org.arquillian.smart.testing.ftest.testbed.configuration.Mode.SELECTING;
+import static org.arquillian.smart.testing.ftest.testbed.configuration.Strategy.AFFECTED;
+import static org.arquillian.smart.testing.ftest.testbed.configuration.Strategy.FAILED;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class LocalChangesFailedTestsSelectionExecutionFunctionalTest {
```

Comment:
Do we really need this test? It seems similar to the previous. WDYT @lordofthejars